packages/components/src/components/link-tile/link-tile.tsx

```diff
@@ -1,54 +1,43 @@
+import type { BreakpointCustomizable, SelectedAriaAttributes, PropTypes } from '../../types';
+import type {
+  LinkTileSize,
+  LinkTileWeight,
+  LinkTileAspectRatio,
+  LinkTileAlign,
+  Tile,
+  LinkTileWeightDeprecated,
+} from '../../utils';
+import type { LinkTileAriaAttribute, LinkTileTarget } from './link-tile-utils';
 import { Component, Element, h, Prop } from '@stencil/core';
 import {
   AllowedTypes,
   attachComponentCss,
   getPrefixedTagNames,
   parseJSON,
   validateProps,
+  throwIfAlignTopAndNotCompact,
   warnIfDeprecatedPropValueIsUsed,
+  LINK_TILE_WEIGHTS,
 } from '../../utils';
 import { getComponentCss } from './link-tile-styles';
-import type { BreakpointCustomizable, SelectedAriaAttributes, PropTypes } from '../../types';
-import type { LinkAriaAttribute } from '../link/link-utils';
 import { LINK_ARIA_ATTRIBUTES } from '../link/link-utils';
-import type {
-  LinkTileAriaAttribute,
-  LinkTileAspectRatio,
-  LinkTileAlign,
-  LinkTileWeight,
-  LinkTileSize,
-  LinkTileWeightDeprecated,
-  LinkTileTarget,
-} from './link-tile-utils';
-import {
-  LINK_TILE_ALIGNS,
-  LINK_TILE_ASPECT_RATIOS,
-  LINK_TILE_SIZES,
-  LINK_TILE_WEIGHTS,
-  throwIfAlignTopAndNotCompact,
-} from './link-tile-utils';
+import { sharedTilePropTypes } from './link-tile-utils';
 
 const propTypes: PropTypes<typeof LinkTile> = {
-  size: AllowedTypes.breakpoint<LinkTileSize>(LINK_TILE_SIZES),
+  ...sharedTilePropTypes,
   weight: AllowedTypes.breakpoint<LinkTileWeight>(LINK_TILE_WEIGHTS),
-  aspectRatio: AllowedTypes.breakpoint<LinkTileAspectRatio>(LINK_TILE_ASPECT_RATIOS),
-  label: AllowedTypes.string,
-  description: AllowedTypes.string,
-  align: AllowedTypes.oneOf<LinkTileAlign>(LINK_TILE_ALIGNS),
-  gradient: AllowedTypes.boolean,
-  compact: AllowedTypes.breakpoint('boolean'),
   href: AllowedTypes.string,
   target: AllowedTypes.string,
   download: AllowedTypes.string,
   rel: AllowedTypes.string,
-  aria: AllowedTypes.aria<LinkAriaAttribute>(LINK_ARIA_ATTRIBUTES),
+  aria: AllowedTypes.aria<LinkTileAriaAttribute>(LINK_ARIA_ATTRIBUTES),
 };
 
 @Component({
   tag: 'p-link-tile',
   shadow: { delegatesFocus: true },
 })
-export class LinkTile {
+export class LinkTile implements Tile {
```

Comment:
I think we need another name for `Tile` here.  Either `ITile` since `I` is a common prefix for interfaces or `ITileProps` or `ISharedTileProps` since props is the only content of this interface/type.